iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/compaction/IcebergTableOptimizer.java

```diff
@@ -115,21 +128,11 @@ private List<org.apache.hadoop.hive.common.TableName> getTables() {
     }
   }
 
-  private List<String> getPartitions(org.apache.iceberg.Table icebergTable) {
+  private org.apache.hadoop.hive.ql.metadata.Table getHiveTable(String dbName, String tableName) {
```

Comment:
static

Reply:
i would still use static and pass conf as param: `resolveTable(conf, dbName, tableName)`